resources/js/home.js

```diff
@@ -0,0 +1,13 @@
+var selectedItems = [];
```

Comment:
Is there any reason to store the script file in a separate folder? Especially when there's just a single one